google/auth/metrics.py

```diff
@@ -123,6 +125,15 @@ def reauth_continue():
     return "{} {}".format(python_and_auth_lib_version(), REQUEST_TYPE_REAUTH_CONTINUE)
 
 
+# x-goog-api-client header value for BYOID calls to the sts endpoint.
+# Example: "gl-python/3.7 auth/1.1 google-byoid-sdk source/aws sa-impersonation/True sa-impersonation/True"
+def byoid_metrics_header(metrics_options):
```

Comment:
Does this function have a unit test? It would be useful to add one that makes sure a whitespace is not appended

Reply:
added test